src/Core/Cart/Calculator.php

```diff
@@ -117,16 +129,16 @@ public function addCartRule(CartRuleData $cartRule)
     /**
      * run the whole calculation process: calculate rows, discounts, fees.
      *
-     * @param int $computePrecision
+     * @param int $computePrecision Not used since 1.7.7.0, kept for backward compatibility
```

Comment:
Besides you don't use the parameter any more at all, so it's completely ignored this looks like a BC break since you can't force the precisions via this parameter any more
You should check if the value is null, and use the class field as the default fallback in that case

Reply:
@jolelievre I don't think we can add `@deprecated` to a function's param 🤔.

I don't think it's a BC break, the computing precision was used in a sub calculation of fees that we never return as it. Plus, we should use the same precision for all calculations if we want to have coherent results.